decisions/0004-planning-of-developments.md

```diff
@@ -2,72 +2,82 @@
+
+### Multi-Step Solution Including Website, Readme.txt and Discussions (version of the 6th of March 2023)
+
 The multi-step solution includes the following points:
- * Create a New Developer Guide on the MP Docs page that describes, at a high level, how codes work together, what is expected of new PRs, and who to contact.
- * Update to MP Website and Open Source Software Section with link to New Developer Guide
- * Update to Readme.txt of individal repos with link to New Developer Guide
- * Create a GitHub discussions board in MP Foundation repo where developers could discuss/announce new additions. Link to this from the New Devleoper Guide
+
+* Create a New Developer Guide on the MP Docs page that describes, at a high level, how codes work together, what is expected of new PRs, and who to contact.
+* Update to MP Website and Open Source Software Section with link to New Developer Guide
+* Update to Readme.txt of individal repos with link to New Developer Guide
+* Create a GitHub discussions board in MP Foundation repo where developers could discuss/announce new additions. Link to this from the New Devleoper Guide
 The following points are the summary of our discussion:
- * Good as it would help developers to get started
- * Good as it would point developers of all codes to one guide and they would understand how the codes work together
- * Good developers would only have one place to look
- * Bad because MP Foundation repo might not attract many users
+* Good as it would help developers to get started
+* Good as it would point developers of all codes to one guide and they would understand how the codes work together
+* Good developers would only have one place to look
+* Bad because MP Foundation repo might not attract many users
+
+### Multi-Step Solution Including Website, Readme.txt and Discussions (version of the 1st of May 2023)
 
-**Multi-Step Solution Including Website, Readme.txt and Discussions (version of the 1st of May 2023):**
 The multi-step solution includes the following points:
- * Create a New Developer Guide on the MP Docs page that describes, at a high level, how codes work together, what is expected of new PRs, and who to contact.
- * Update to MP Website and Open Source Software Section with link to New Developer Guide
- * Update to Readme.txt of individal repos with link to New Developer Guide
- * Optional:  Create a GitHub discussions board in MP Foundation repo where developers could discuss/announce new additions. Link to this from the New Devleoper Guide
- * Optional: Have a "Dev Project" discussion board in every single MP repo with a pinned comment at the top linking the "MP Dev Guide"
+
+* Create a New Developer Guide on the MP Docs page that describes, at a high level, how codes work together, what is expected of new PRs, and who to contact.
+* Update to MP Website and Open Source Software Section with link to New Developer Guide
+* Update to Readme.txt of individal repos with link to New Developer Guide
```

Comment:
I think the link to the dev guide was meant for PR templates (i.e. check this box to confirm you're read our dev guide), not readmes.

Reply:
I remember that we also discussed readme's during our last meeting. I personally would probably also first look at the readme before raising a PR.
And you are not covering the cases where people do not even consider that PR's are actually welcome.

Reply:
Good point. I like readme too. :)

Reply:
We need to add the PR templates then, right?

Reply:
I think you're right in that readmes are more often consulted whereas PR templates are often ignored. So maybe we skip the templates.